Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM node:16.18.0
+# Deprecated Dockerfile used for Heroku Deployment 
+# Kept around if someone comes looking to deploy Docs using a docker image
 
-WORKDIR /src
+# FROM node:16.18.0
 
-COPY package.json package-lock.json ./
-RUN npm ci
+# WORKDIR /src
 
-COPY . .
-RUN npm run build
+# COPY package.json package-lock.json ./
+# RUN npm ci
 
-EXPOSE 3000
+# COPY . .
+# RUN npm run build
 
-ENTRYPOINT ["/usr/local/bin/npm"]
-CMD ["start"]
+# EXPOSE 3000
+
+# ENTRYPOINT ["/usr/local/bin/npm"]
+# CMD ["start"]
```

Comment:
I would rather delete this file, or uncomment the lines. It serves no purpose in this state.

> Kept around if someone comes looking to deploy Docs using a docker image

In that case the lines should be uncommented.